modules/ROOT/pages/provisioning-oraclecloud.adoc

```diff
@@ -0,0 +1,326 @@
++
+.Mark as compatible with aarch64 VM shapes
+[source, bash]
+----
+shapes_aarch64=(
+    "VM.Standard.A1.Flex"
+    "VM.Standard.A2.Flex"
+)
+for shape in "${shapes_aarch64[@]}"; do
+    oci compute image-shape-compatibility-entry add --image-id "${image_id}" --shape-name "${shape}"
+done
+----
++
+. To list all the compatible shapes for an image:
++
+[source, bash]
+----
+oci compute image-shape-compatibility-entry list --image-id "${image_id}"
+----
+
+== Launching an instance
+
+. Create a Virtual Cloud Network:
++
+[source, bash]
+----
+vcn_id="$(oci network vcn create \
+    --compartment-id "${compartment_ocid}" \
+    --cidr-blocks '["10.0.0.0/16"]' \
+    --display-name "fedora-coreos-vcn" \
+    --dns-label "fcos.example.com" \
+    --wait-for-state AVAILABLE \
+    | jq -r '.data.id')"
+----
++
+. Add a subnet:
++
+[source, bash]
+----
+subnet_id="$(oci network subnet create \
+    --cidr-block "10.0.0.0/24" \
+    --compartment-id "${compartment_ocid}" \
+    --vcn-id "${vcn_id}" \
+    --display-name "fedora-coreos-subnet" \
+    --dns-label "main" \
+    --wait-for-state AVAILABLE \
+    | jq -r '.data.id')"
+----
++
+. Create an Internet Gateway:
++
+[source, bash]
+----
+gateway_id="$(oci network internet-gateway create \
+    --compartment-id "${compartment_ocid}" \
+    --vcn-id "${vcn_id}" \
+    --is-enabled true \
+    --display-name "fedora-coreos-gateway" \
+    | jq -r '.data.id')"
+----
++
+. Add a Rule to the Route Table:
++
+[source, bash]
+----
+route_table="$(oci network route-table list \
+    --compartment-id "${compartment_ocid}" \
+    --vcn-id "${vcn_id}" \
+    | jq -r '.data[0].id')"
+
+oci network route-table update \
+    --rt-id "${route_table}" \
+    --route-rules '[{"cidrBlock":"0.0.0.0/0","networkEntityId":"'"${gateway_id}"'"}]' \
+    --force
+----
++
+. Pick an availability domain:
++
+[source, bash]
+----
+availability_domain="$(oci iam availability-domain list | jq -r '.data[0].name')"
+----
++
+. Launch an instance. Your Ignition configuration must be passed to the VM as its user data, or you can skip passing user data if you just want SSH access. This provides an easy way to test out FCOS without first creating an Ignition config.
++
+.Example launching with only SSH keys configured
+[source, bash]
+----
+NAME=fedora-coreos
+SHAPE=VM.Standard.E2.1.Micro
+SSHKEYS="/path/to/authorized_keys" # path to authorized_keys file
+
+oci compute instance launch \
+    --compartment-id "${compartment_ocid}" \
+    --availability-domain "${availability_domain}" \
+    --display-name "${NAME}" \
+    --image-id "${image_id}" \
+    --shape "${SHAPE}" \
+    --subnet-id "${subnet_id}" \
+    --assign-public-ip true \
+    --ssh-authorized-keys-file "${SSHKEYS}"
+----
++
+.Example launching customized Ampere instance
+[source, bash]
+----
+NAME=fedora-coreos
+SHAPE=VM.Standard.A1.Flex
+DISK=50                            # size of boot volume in GBs
+OCPUS=1                            # number of allocated OCPUs
```

Comment:
```suggestion
OCPUS=2                            # number of allocated OCPUs
```

Maybe we should use at least 2 here. I've seen some weird behavior before when only 1 CPU is available in VMs because some processes assume multithreading and a single CPU can get deadlocked.